cypress/integration/App-spec.js

```diff
@@ -0,0 +1,338 @@
+describe('the main user flow', () => {
+  it('should visit the page with test data', () => {
+    cy
+      .fixture('../fixtures/allMovies.json')
+      .then(data => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+          statusCode: 200, 
+          body: data
+        })
```

Comment:
Glad we verified our intercepts are working currently with changing a value in the dummy data.